2200040/직사각형 길이 찾기.py

```diff
@@ -0,0 +1,20 @@
+# 대칭만들기
+# 나머지 변의 길이 출력
+
+T = int(input())
+
+for t in range(1, T+1):
+    a, b, c = list(map(int, input().split()))
+    d = 0
+    if a==b:
+        d = c
+        print(f'#{t} {d}')
+    elif b==c:
+        d = a
+        print(f'#{t} {d}')
+    elif a==c:
+        d = b
+        print(f'#{t} {d}')
+    elif a==b==c:
+        b = a
+        print(f'#{t} {d}')
```

Comment:
```suggestion
    if a==b:
        d = c
    elif b==c:
        d = a
    elif a==c:
        d = b
    elif a==b==c:
        b = a
    print(f'#{t} {d}')
```
숫자들을 비교해 정답을 잘 찾아주셨네요 😄 
if문 에서 정답을 출력할 때 다 같은 방법으로 출력하므로, if문 밖에서 출력하는 방법도 있을 것 같아요!